packages/medusa/src/models/schemas/line-item.js

```diff
@@ -9,7 +9,7 @@ export default new mongoose.Schema({
   thumbnail: { type: String },
   is_giftcard: { type: Boolean, default: false },
   has_shipping: { type: Boolean, default: false },
-
+  is_shipped: { type: Boolean, default: false },
```

Comment:
I think `shipped_quantity` would be better as we may not ship the entire line-item